kubevela/install.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
-helm repo add kubevela https://charts.kubevela.net/core
-helm repo update
-helm install --create-namespace -n vela-system kubevela kubevela/vela-core --wait --version v1.5.2
\ No newline at end of file
+curl -fsSl https://static.kubevela.net/script/install.sh | sudo bash -s 1.5.3
```

Comment:
Usually we do not allow curl but if that is from the maintainer we can give it a pass